src/BrightcovePlayer.js

```diff
@@ -113,6 +113,18 @@ BrightcovePlayer.prototype.seekTo = Platform.select({
     }
 });
 
+BrightcovePlayer.prototype.seekToEnd = Platform.select({
+    ios: function () {
+        NativeModules.BrightcovePlayerManager.seekToEnd(
+            ReactNative.findNodeHandle(this)
+        );
+    },
+    android: function () {
+        // Not implemented for Android
```

Comment:
Since you aren't calling it from the JS code, may be better to throw an error here. But who am I to judge? ;-)